core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -1922,9 +1922,8 @@ public DirectoryBrowserSupport doWs( StaplerRequest req, StaplerResponse rsp ) t
      */
     public HttpResponse doDoWipeOutWorkspace() throws IOException, ServletException, InterruptedException {
         checkPermission(Functions.isWipeOutPermissionEnabled() ? WIPEOUT : BUILD);
-        R b = getSomeBuildWithWorkspace();
-        FilePath ws = b!=null ? b.getWorkspace() : null;
-        if (ws!=null && getScm().processWorkspaceBeforeDeletion(this, ws, b.getBuiltOn())) {
```

Comment:
Whatever else the virtues of this PR (and a test proving what it fixes would be desirable), the removal of the call to `processWorkspaceBeforeDeletion` means it is causing _some_ regression.